### PR TITLE
added photo scanning page

### PR DESCRIPTION
**Pull request by skellyska**
added page for photo scanning. Page displays picture after captured with mobile device camera.

### CODE REVIEW

File: src/Components/LandingPage.js [hunk 1]
```diff
@@ -1,10 +1,17 @@
 import React from 'react';
+import { Link } from 'react-router-dom';
 
 function LandingPage () {
     return (
         <div>
             <h1>Welcome to Litter.ai</h1>
             <p>With just a click, sort your garbage and save the world</p>
+            <div style={{ position: 'fixed', bottom: '20px', left: '50%', transform: 'translateX(-50%)', width: '200px' }}>
```
Comment (JMStudiosJoe) [inline src/Components/LandingPage.js:9]: lets make a css file and this should also be in its own directiory. src/components/landingPage/LandingPage.js and another file in there LandingPage.css that the js file imports. That is where all styles for this component should go.

File: src/Components/script.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+document
+  .getElementById("cameraFileInput")
```
Comment (JMStudiosJoe) [inline src/Components/script.js:2]: lets call this file somehting other than script.js something more relevent to what it is doing

File: src/Components/LandingPage.js (hunk 1 above)
Comment (JMStudiosJoe) [reply src/Components/LandingPage.js:9]: was the styling added?

File: src/Components/ImageCapture.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+document
```
Comment (JMStudiosJoe) [inline src/Components/ImageCapture.js:1]: lets change this to a clickable callable function maybe. This is pretty old style Javascript, it "works" but we can be cleaner :)

**Review by JMStudiosJoe — CHANGES_REQUESTED** [review]

File: src/Components/LandingPage.js (hunk 1 above)
Comment (skellyska) [reply src/Components/LandingPage.js:9]: I didn't add any styling. I'm going to add the styling on a separate PR

File: src/Components/ImageCapture.js (hunk 3 above)
Comment (skellyska) [reply src/Components/ImageCapture.js:1]: Ok, I will change and re-submit

Comment (JMStudiosJoe) [reply src/Components/ImageCapture.js:1]: For now yes, we can find the better way to do this when needed. Pretty sure the correct way will be a useEffect function

Comment (JMStudiosJoe) [reply src/Components/ImageCapture.js:1]: yes, this should be a useEffect function within the camera capture component

Comment (JMStudiosJoe) [reply src/Components/ImageCapture.js:1]: we can go over the changes in another pr as well for optimizations

**Review by JMStudiosJoe — APPROVED** [review]

**Pull request merged**